gtest/src/program.rs

```diff
@@ -434,15 +448,43 @@ impl<'a> Program<'a> {
     /// Reads the program’s transformed state as a byte vector. The transformed
     /// state is a result of applying the `fn_name` function from the `wasm`
     /// binary with the optional `argument`.
-    pub fn read_state_bytes_using_wasm(
+    ///
+    /// # Usage
+    /// You can pass arguments as `Option<(arg1, arg2, ...)>` or by using
+    /// [`read_state_wasm_args`] macro.
+    ///
+    /// # Examples
+    ///
+    /// Read state bytes with no arguments passed to wasm.
+    /// ```ignore
```

Comment:
`ignore`d examples have a risk of being stale. I'd replace several examples with one compilable example.